android/src/main/java/com/swmansion/gesturehandler/core/GestureUtils.kt

```diff
@@ -54,23 +54,8 @@ object GestureUtils {
   fun coneToDeviation(angle: Double): Double =
     cos(Math.toRadians(angle / 2.0))
 
-  fun updateStylusData(event: MotionEvent): StylusData {
-    // On web and iOS 0 degrees means that stylus is parallel to the surface. On android this value will be PI / 2.
-    val altitudeAngle = (PI / 2) - event.getAxisValue(MotionEvent.AXIS_TILT).toDouble()
-    val pressure = event.getPressure(0).toDouble()
-
-    val orientation = event.getOrientation(0).toDouble()
-
-    // To get azimuth angle, we need to use orientation property (https://developer.android.com/develop/ui/compose/touch-input/stylus-input/advanced-stylus-features#orientation).
-    val azimuthAngle = (orientation + PI / 2).mod(2 * PI)
-
-    val tilts = spherical2tilt(altitudeAngle, azimuthAngle)
-
-    return StylusData(tilts.first, tilts.second, altitudeAngle, azimuthAngle, pressure)
-  }
-
   // Source: https://w3c.github.io/pointerevents/#converting-between-tiltx-tilty-and-altitudeangle-azimuthangle
-  private fun spherical2tilt(altitudeAngle: Double, azimuthAngle: Double): Pair<Double, Double> {
+  fun spherical2tilt(altitudeAngle: Double, azimuthAngle: Double): Pair<Double, Double> {
```

Comment:
I think this can also be moved to companion of `StylusData` and be kept private

Reply:
At the moment I cannot see where we would want to use it besides stylus. We can always move it out when needed but I doubt we will move it in if we merge it like this 😅.

Reply:
Okay, I agree. Done in [39b2d10](https://github.com/software-mansion/react-native-gesture-handler/pull/3111/commits/39b2d10e4d1b31969f56009f8dabb9856119e257)